src/components/ContactCard/ContactCard.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from 'next/link';
+import Divider from '../Divider/Divider';
+import styles from './ContactCard.module.scss';
+
+const ContactCard = () => {
+  return (
+    <div>
+      <div className={styles.languageToggle}>
+        <div className={styles.swedish}>
+          <button className={styles.button}>
+            <picture className={styles.flag}>
+              <img src="/sweden.svg" alt="Swedish flag"/>
+            </picture>
+            <p>Svenska</p>
+          </button>
+        </div>
+        <div className={styles.english}>
+          <button className={styles.button}>
+            <picture className={styles.flag}>
+              <img src="/uk.svg" alt="English flag"/>
+            </picture>
+            <p>English</p>
+          </button>
+        </div>
```

Comment:
Make this into an anchor, a button uses the browser's default sans-serif font
```suggestion
        <a className={styles.button}>
          <picture className={styles.flag}>
            <img src="/sweden.svg" alt="Swedish flag"/>
          </picture>
          <p>Svenska</p>
        </a>
        <a className={styles.button}>
          <picture className={styles.flag}>
            <img src="/uk.svg" alt="English flag"/>
          </picture>
          <p>English</p>
        </a>
```